nx-cloud/templates/cloud.yml

```diff
@@ -402,6 +428,44 @@ spec:
           {{ end }}
         {{ end }}
 
+
+    {{ if .Values.secret.name }}
+      {{ if .Values.secret.githubAuthToken }}
+            - name: NX_CLOUD_GITHUB_AUTH_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secret.name }}
+                  key: {{ .Values.secret.githubAuthToken }}
+      {{ end }}
+      {{ if .Values.secret.githubAppId }}
+            - name: NX_CLOUD_GITHUB_APP_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secret.name }}
+                  key: {{ .Values.secret.githubAppId }}
+      {{end}}
+      {{ if .Values.secret.githubPrivateKey }}
+            - name: NX_CLOUD_GITHUB_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secret.name }}
+                  key: {{ .Values.secret.githubPrivateKey }}
+      {{ end }}
+    {{ else }}
+      {{ if .Values.secret.githubAuthToken }}
+            - name: SECRET_FILE_NX_CLOUD_GITHUB_AUTH_TOKEN
+              value: {{ .Values.secret.githubAuthToken }}
+      {{ end }}
+      {{ if .Values.secret.githubAppId }}
+            - name: SECRET_FILE_NX_CLOUD_GITHUB_APP_ID
+              value: {{ .Values.secret.githubAppId }}
+      {{ end }}
+      {{ if .Values.secret.githubPrivateKey }}
+            - name: SECRET_FILE_NX_CLOUD_GITHUB_PRIVATE_KEY
+              value: {{ .Values.secret.githubPrivateKey }}
+      {{ end }}
+    {{ end }}
```

Comment:
No changes, just moved out of GH configuration block since we always want to pass these values if they are available